jake/iq/iq.py

```diff
@@ -78,6 +79,10 @@ def get_report_url(self):
     """gets report url from IQ Server result"""
     return self._report_url
 
+  def get_absolute_report_url(self):
+    """get full (non-relative) url to IQ Server report"""
+    return urllib.parse.urljoin(self._iq_url, self._report_url)
```

Comment:
I think you are good to merge, but just asking, urljoin will just ignore joining if the URL is already absolute? If so that's cool! (Asking to stuff into my head for a future time where I likely will never need to know this)

Reply:
Excellent question (and I only know the answer after a bunch of searching to find the magic method). I went ahead and added a comment to make it clearer which behavior we require. 30d60f7

And yes, if the second arg is absolute, it will be used.
https://docs.python.org/3/library/urllib.parse.html